doc/sailfish-webview-webenginesettings.cpp

```diff
@@ -212,4 +229,15 @@ void SailfishOS::WebEngineSettings::enableLowPrecisionBuffers(bool enabled);
 */
 void SailfishOS::WebEngineSettings::setPreference(const QString &key, const QVariant &value);
```

Comment:
The changes here are fine, but I think line 206/223 of the original description of `setPreference` is redundant (since the same link to `WebEngineSettings` is already provided on the line above). This might be a good opportunity to remove it.

The link to the MDM docs is also unfortunately broken, so probably not worth keeping.

Reply:
Added a commit on top that cleans them up.